css-two-column-layout/styles.css

```diff
@@ -0,0 +1,101 @@
+.container {
+  width: 1094px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-three-quarters {
+  flex-basis: 75%;
+}
+
+/* .column-half {
+  flex-basis: 50%;
+} */
```

Comment:
Why is this commented out? I would think it is necessary 🤔

Reply:
@codingcodymiller marking yout here to let you know that I made the changes